### PR TITLE
Use split.js for splitting code and web output

### PR DESCRIPTION
**Pull request by johnpryan**
copies split.dart file from flutter/devtools repo

### CODE REVIEW

File: web/experimental/embed-new.html [hunk 1]
```diff
@@ -28,6 +28,7 @@
     <link href="../packages/codemirror/addon/hint/show-hint.css" rel="stylesheet" media="screen">
     <link href="../packages/codemirror/theme/elegant.css" rel="stylesheet" media="screen">
     <script src="../packages/codemirror/codemirror.js" defer></script>
+    <script type="text/javascript" src="packages/split/split.min.js"></script>
```
Comment (devoncarew) [inline web/experimental/embed-new.html]: @jacob314 - given that people are relying on package:split for the js resources, we may consider also publishing the dart code (see https://github.com/dart-lang/dart-pad/pull/1036/files#diff-a9e9818d8a5762e16905000b338d8a9c above).

@johnpryan, if that sounds good, you might track that as a new issue against the devtools repo.

Comment (johnpryan) [reply web/experimental/embed-new.html]: good idea, created https://github.com/flutter/devtools/issues/565 to track

File: web/experimental/embed-new.html [hunk 2]
```diff
@@ -56,8 +57,8 @@
 </div>
 <section id="tab-container" class="flex-auto d-flex flex-column no-overflow">
     <div id="user-code-view" class="flex-auto d-flex flex-row no-overflow">
-        <div id="user-code-editor" class="flex-auto d-flex flex-column" spellcheck="false"></div>
-        <div id="web-output" class="p-1 d-flex" hidden>
+        <div id="user-code-editor" spellcheck="false"></div>
+        <div id="web-output" hidden>
```
Comment (RedBrogdon) [inline web/experimental/embed-new.html]: I've been meaning to kill that "p-1" class for a week now. Thanks. 😄

File: test/experimental/new_embed_test.html [hunk 3]
```diff
@@ -28,6 +28,7 @@
     <link href="../packages/codemirror/addon/hint/show-hint.css" rel="stylesheet" media="screen">
     <link href="../packages/codemirror/theme/elegant.css" rel="stylesheet" media="screen">
     <script src="../packages/codemirror/codemirror.js" defer></script>
+    <script src="../packages/split/split.min.js" defer></script>
```
Comment (RedBrogdon) [inline test/experimental/new_embed_test.html]: To make sure this file is accessible when deployed to prod, you'll need to update the regex for packages in `app.yaml`:

https://github.com/dart-lang/dart-pad/blob/master/web/app.yaml

Comment (johnpryan) [reply test/experimental/new_embed_test.html]: cool, fixed in 6e71d20

**Comment by johnpryan** [comment]
Screen Shot 2019-04-23 at 10 38 10 AM

File: lib/experimental/split.dart [hunk 4]
```diff
@@ -0,0 +1,127 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Thin JS interop wrapper around https://split.js.org/
```
Comment (domesticmouse) [inline lib/experimental/split.dart]: Can we work with @devoncarew's team to migrate this file into the split package? It makes sense to me to have this matched with the JS library it is wrapping.

Comment (devoncarew) [reply lib/experimental/split.dart]: I sure we'd accept PRs into https://github.com/flutter/devtools/tree/master/third_party/packages to do that -

Comment (johnpryan) [reply lib/experimental/split.dart]: sounds good, PR [here](https://github.com/flutter/devtools/pull/572)

Comment (johnpryan) [reply lib/experimental/split.dart]: discussed offline with Andrew: we're going to merge as-is and incorporate the new version of the split package later

**Comment by johnpryan** [comment]
discussed offline with Andrew: we're going to merge as-is and incorporate the new version of the split package later

**Pull request merged**